providers/apple/session.go

```diff
@@ -47,6 +49,8 @@ type IDTokenClaims struct {
 	jwt.StandardClaims
 	AccessTokenHash string `json:"at_hash"`
 	AuthTime        int    `json:"auth_time"`
+	Email           string `json:"email"`
+	IsPrivateEmail  bool   `json:"is_private_email"`
```

Comment:
I got around to testing this today and I had to change this to `json:"is_private_email,string"`

Reply:
Great. Love a good boolean as a string. I'll make the change.